tests/test_correlation.py

```diff
@@ -133,3 +144,38 @@ def test_cn_unequal_series_corr():
     sr_b = pd.Series([100, 200, 99, 101, 201, 199, 299, 300, 301, 500, 501, 505, 10, 12, 1001, 1050])
 
     assert distance_cn_correlation(sr_a, sr_b) > 0.7
+
+
```

Comment:
Stress test to make sure correlation matrix generates properly without the encoding step.